ceph/ceph_admin/dashboard.py

```diff
@@ -112,3 +113,39 @@ def validate_enable_dashboard(cls, username, password):
         LOG.info(
             "Dashboard API login is successful, status code : %s" % resp.status_code
         )
+
+
+def enable_monitoring(cls):
+    """Method to enable the monitoring stack module.
+
+    if user bootstrap with skip-dashboard option
+    then enabling the monitoring stack module.
+
+    Args:
+        cls (CephAdmin object) : cephadm instance object.
+    """
+    out, _ = cls.shell(args=["ceph", "mgr", "services"])
+    formatted = json.loads(out)
+
+    # Getting the ip address to set monitoring stack URL's
+    url = formatted.get("prometheus")
+
+    # Finding IP address using IPV4 format
+    pattern = re.compile(r"(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})")
+    ip = pattern.findall(url)[0]
+
+    # To enable monitoring stack
+
+    MONITORING_ENABLE_COMMANDS = [
```

Comment:
as per above comment, would it make sense to split this up into individual functions?

Reply:
same as above, I think it's not needed